src/pyxplor/pyxplor.py

```diff
@@ -0,0 +1,62 @@
+def plot_binary(input_df: pd.DataFrame,
+                list_of_variables: list,
+                orientation: str,
+                fig_width: int,
```

Comment:
Do we want default width and heights? I know @arturoboquin suggested 10 x 6 in his function

Reply:
Good question. I think at this point we are expected to have a general idea, and then when we write the function and create some plots we will have a better understanding of that. It might vary depending on the type of data.

Reply:
```suggestion
```